docs/resources/dataarts_sudio_catalog.md

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "DataArts Studio"
+---
+
+# huaweicloud_dataarts_studio_catalog
+
+Manages a DataArts Studio catalog resource within HuaweiCloud.
+
+## Example Usage
+
+### Create a root catalog
+
+```hcl
+variable "workspace_id" {}
+variable "catalog_name" {}
+variable "catalog_owner" {}
+
+resource "huaweicloud_dataarts_studio_catalog" "test-root" {
+  workspace_id = var.workspace_id
+  name         = var.catalog_name
+  owner        = var.catalog_owner
+}
+```
+
+### Create a subordinate catalog
+
+```hcl
+variable "workspace_id" {}
+variable "catalog_name" {}
+variable "catalog_owner" {}
+variable "parent_catalog_id" {}
+
+resource "huaweicloud_dataarts_studio_catalog" "test-sub" {
+  workspace_id = var.workspace_id
+  name         = var.catalog_name
+  owner        = var.catalog_owner
+  parent_id    = var.parent_catalog_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `workspace_id` - (Required, String, ForceNew) Specifies the ID of DataArts Studio workspace.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the name of catalog.
+
+* `owner` - (Required, String) Specifies the name of person responsible for catalog. The responsible person must exist
+  in the system.
+
+* `description` - (Optional, String) Specifies the description of catalog.
+
+* `parent_id` - (Optional, String) Specifies the parent catalog ID of catalog.  
+  It's **Required** when you create a subordinate catalog, and directory level cannot exceed `3` layer.
+
+* `prev_id` - (Optional, String) Specifies the ID of the previous node in the catalog.  
+  When querying the catalog, if the field is null, it means that is the first node in the current catalog directory.
+
+* `next_id` - (Optional, String) Specify the ID of the next node in the catalog.  
+  When querying the catalog, if the field is null, it means that is the last node in the current catalog directory.
+
+-> **NOTE:** The `prev_id` and `next_id` will change with the position of the catalog in the current directory.
```

Comment:
```suggestion
-> The `prev_id` and `next_id` will change with the position of the catalog in the current directory.
```
`->` and `-> **Note**:` are same.

`->` means note (with blue)
`~>` means note (with yellow)
`!>` means warning

Reply:
ok, fixed.